src/middlewared/middlewared/etc_files/rancher/k3s/flags.py

```diff
@@ -50,3 +51,9 @@ def render(service, middleware):
             'protect-kernel-defaults': True,
             'disable': [] if config['servicelb'] else ['servicelb'],
         }))
+
+    with open('/etc/containerd.json', 'w') as f:
```

Comment:
Out of curiosity, what consumes this file?

Reply:
Containerd consumes this file where we tell it to validate host paths or not